src/glang/parser/parser.c3

```diff
@@ -606,6 +636,15 @@ function parse_struct_type : (cursor : Cursor mut&) -> StructType = {
     return {members, meta(start_offset, cursor.position)};
 }
 
+function parse_function_type : (cursor : Cursor mut&) -> FunctionType = {
+    let start_offset : isize = cursor.position;
+
+    let args : Vector<Type> = cursor:parse_type_list_between(sv("("), sv(")"));
+    runtime_assert(cursor:try_skip_next_and_surrounding_white_space("->"));
```

Comment:
Shouldn't this be an actual syntax error?

Reply:
Whoops, forgot we actually supported this.